src/MatrixClient.ts

```diff
@@ -677,6 +679,23 @@ export class MatrixClient extends EventEmitter {
         });
     }
 
+        /**
+     * Gets the joined members in a room. The client must be in the room to make this request.
+     * @param {string} roomId The room ID to get the joined members of.
+     * @param {string} membership Get members who are of this type. Leave undefined if no filtering is needed.
+     * @param {string} notMembership Do not get members who are of this type. Leave undefined if no filtering is needed.
+     * @returns {Promise<string>} The joined user IDs in the room
+     */
+    @timedMatrixClientFunctionCall()
+    public getRoomMembers(roomId: string, membership?: MembershipEnum, notMembership?: MembershipEnum) {
+        return this.doRequest("GET", "/_matrix/client/r0/rooms/" + encodeURIComponent(roomId) + "/members", {
+            membership,
+            not_membership: notMembership,
+        }).then(response => {
+            return response['chunk'] as MatrixRoomMemberEvent[];
```

Comment:
is it a string or a structure? If it's a structure, I have quite a few concerns.